scripts/sct_label_vertebrae.py

```diff
@@ -849,7 +849,7 @@ def label_segmentation(fname_seg, list_disc_z, list_disc_value, fig_anat_straigh
     """
 
     if fig_anat_straight is None:
-        fig_anat_straight = Param.fig_anat_straight
+        fig_anat_straight = Param().fig_anat_straight
```

Comment:
You are right... That why I had it as a class variable first and used the class everywhere. I could initialize param if it does not exist already... ore keep it that way, not sure.